### ISSUE TITLE
Update Technique G195 procedure

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1931]
https://w3c.github.io/wcag/techniques/general/G195

I believe the first two steps of the test procedure need to be updated, with an added preamble, to read:
> For each user interface element on the page that should receive keyboard focus: 
>1. Navigate to the element and check that it has a visible focus indicator.
>1. Check that the focus indicator area…

Step 3 in the old version becomes step 2 here with the remaining 2 steps becoming steps 3 and 4. Revised version to read:
> For each user interface element on the page that should receive keyboard focus: 
>1. Navigate to the element and check that it has a visible focus indicator.
>1. Check that the focus indicator area is at least the size of a 1 CSS px border around the component.
>1. Check that the change of contrast of the indicator between focused and unfocused states has a ratio of 3:1 for the minimum focus indicator area.
>1. If the focus indicator does not have 3:1 contrast ratio with its adjacent colors, check that it is at least 2px thick.

I think step 1 covers ‘partially visible' as well (and Focus Visible too, hmmm). However, there is a second challenge with this test, in that it doesn’t check that the focus indicator achieves 3:1 against the background (which is required for 1.4.11 Non-text Contrast). Not sure whether we should even go there… Kinda underscores how overlapped these SCs are. It may be a case of making 2 or 3 separate techniques plus the Focus Visible one, making them combine to achieve this. Similar to how [H49](https://www.w3.org/WAI/WCAG21/Techniques/html/H49) is dealt with in 1.3.1
> This technique relates to Success Criterion 1.3.1: Info and Relationships (Sufficient, together with G115: Using semantic elements to mark up structure).

**Comment by alastc** [comment]
G195 was also references from focus-appearance-enhanced, so I've separated off a variation of the technique for that.
